### ISSUE TITLE
Fan control doesnt start with windows

### ISSUE THREAD

**Issue by Skullywag** [Rem0o/FanControl.Releases#1897]
I have to start it manually everytime, there doesnt seem to be a scheduled task in task scheduler for it. I have saved the setting to start with windows etc but doesnt have any effect. Not a big deal just slightly annoying. Is there anything I can check to work out why this is happening?

**Comment by Rem0o** [comment]
What delay have you set under the checkbox? Try with more.

If you manually start the scheduled task, does FanControl opens?

**Comment by Faggi11** [comment]
@Skullywag is it like this => https://github.com/Rem0o/FanControl.Releases/discussions/1939#discussion-5429114
i just tried what Remo said, longer delay at startup, but doesnt help (up to 45sec)

**Comment by p45rJv** [comment]
Looking over the issue I noticed that FanControl is creating a scheduled task with a configuration thats assuming: 
- Every account is an administrator account.
- Administrator account will log in at startup.
- Privileges are escalated by the same account (UAC), as the account logged in. 

You can manually trigger this task if you launch Task Scheduler with privileges given by an admin account (mmc).

The user thats logged in and actually clicking the setting to start FanControl with Windows is not the one FanControl creates the task for. Its created for the account that authenticated UAC it seems. 

This can be reproduced on any multi account setup where the logged in user is not the one giving the privileges.

**Comment by Faggi11** [comment]
Puuuh, i dont know, if i understand that correctly (non-native english here), but since v166 i think, it worked if you trigger **_any user logged in_** in the task, then FC starts w/o UAC with a restricted account, **_but_** the GUI couldnt show up, and if you start manually you get of course, an error, 'cause FC is already running.

**Comment by p45rJv** [comment]
No need to apologize, I'm not the best at expressing myself :P 

1. Open notepad

```powershell
Start-Process PATH\TO\FanControl\FanControl.exe -Verb runAs -Wait
```
2. Select "all files" from the "save as type" dropdown box.
3. Give the script a name and ".ps1" as extension. (For example, FanControlStartup.ps1". 

You have to change the executionpolicy to allow powershell scripts to run. As FanControl needs privileges, I find it better to allow script to be executed than bypass UAC. -For my setup

1. Start powershell as administrator (type cmd, right click, run as admin then write powershell and press enter)
2. Set-ExecutionPolicy Unrestricted (this is permanent)

Then you just have to add the script to startup:
1. win+r 
2. type shell:startup
3. right click and select "create new shortcut) in the file explorer window that opens. 
4. type: 
```
PowerShell "& 'PathToFile\script.ps1'"
```
FanControl will now start when your user logs in, and prompt you for UAC credentials. There are ways to bypass this.

That being said, Its probably for the best that FanControl doesnt allow normal users access to the application. FanControl would still need to change the configuration of the scheduled task to not require log in by an administrator to run. That would resolve the issue. Those that need access to FanControl gui from normal user accounts can mitigate by my approach, or numerous other ways to do it. Its probably best that FanControl doesnt do this. 

tl;dr: The scheduled task created by FanControl could be changed to not require login. Or an option for both in settings. Should resolve the issue for most.

**Comment by Faggi11** [comment]
Okay, thank you very much, i try that tomorrow, then i write a short answere here ;)

(btw. untill v162 or v163 it worked automatically, if you ticked the "Start with Windows checkbox", that you get the UAC prompt)

**Comment by Faggi11** [comment]
Before i post my first Post here, i emailed him already about this, and a few days later we get new version of it, there was the one or other that could be a fix or so, so i wait for it.
But about your PS Script, that is working fine (like before V162 or so), except one little thing, the script windows from the console stays open.
I googled for that and i find the one or other hint and possible solution, but it seems very complicated in some way ;)
I not sooo imortant, just click the "X" and its done.

**Comment by p45rJv** [comment]
```powershell
Start-Process A:\PATH\To\Fancontrol\FanControl.exe -Verb runAs  -WindowStyle Minimized **-ArgumentList '/c'**
```

That should hide it all :)

**Comment by Faggi11** [comment]
Okay thanks, but it doesnt start now, .... i'm going back to previous arguments

**Comment by p45rJv** [comment]
You only need to add "-ArgumentList '/c'". The ' is important

**Comment by Faggi11** [comment]
Oh, i take it from my notification email from your post, there are 2 ** for and after "-ArgumentList '/c'"
i will retry

**Comment by Faggi11** [comment]
Ok, it works, but it's not minimize, and i set it to off in FC, as you said

**Comment by p45rJv** [comment]
Paste the full command you use here

**Comment by Faggi11** [comment]
Start-Process "H:\002_Proggis\System-Tools\Tools & Tuning\FanControl\FanControl.exe" -Verb runAs -WindowStyle Minimized -ArgumentList '/c'

**Comment by p45rJv** [comment]
It seems fine, I cant reproduce that here on 3 different computers. 

Did you close all FC processes before running the script? 

Anywho, you might try right clicking the fc.exe and force it to run minimized in properties. Wild guess though :|

**Comment by Unprocessable** [comment]
Linking similar discussion: #1202

**Comment by Faggi11** [comment]
yeah, kind of ...... i'm happy so far with the solution obove
